src/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFruControlMIB.py

```diff
@@ -0,0 +1,99 @@
+import imp
+import re
+import sys
+
+from sonic_ax_impl import mibs
+from ax_interface import MIBMeta, ValueType, MIBUpdater, MIBEntry, SubtreeMIBEntry
+from ax_interface.encodings import ObjectIdentifier
+
+PSU_PLUGIN_MODULE_NAME = 'psuutil'
+PSU_PLUGIN_MODULE_PATH = "/usr/share/sonic/platform/plugins/{}.py".format(PSU_PLUGIN_MODULE_NAME)
+PSU_PLUGIN_CLASS_NAME = 'PsuUtil'
+
+# TODO: Remove once PSU util will be available directly
+sys.path.append("/usr/lib/python2.7/dist-packages/")
```

Comment:
Is this still needed now that you are using the PsuUtil plugin directly?

Reply:
Yes, it's still need to import PSU module